extensions/binding/org.eclipse.smarthome.binding.astro/src/main/java/org/eclipse/smarthome/binding/astro/internal/job/Job.java

```diff
@@ -1,20 +1,26 @@
 /**
- * Copyright (c) 2014-2017 by the respective copyright holders.
+ * Copyright (c) 2017 by the respective copyright holders.
```

Comment:
If we would like to follow that rule we need to change the project specific `Code Style` guidelines and update all other source files.
IMHO this could be discussed in a separate issue.

As long as the `Code Style` for that project uses `mvn license:format` to keep the license header up to date we should follow that rule. Shouldn't we?

https://www.eclipse.org/smarthome/documentation/development/guidelines.html#a-code-style